code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -327,7 +327,7 @@
 			FindTarget()
 
 
-/mob/living/simple_animal/hostile/proc/AttackingTarget()
+/mob/living/simple_animal/hostile/proc/AttackingTarget(atom/A)
```

Comment:
Since you are doing this change, give the variable a better name by default than `A`